core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
```

Comment:
That's fair.  I wouldn't mind looking for the closest one, as again, it's only on user interaction.